hack/verify-gofmt.sh

```diff
@@ -13,11 +8,6 @@ if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.6') && -z "${FORCE_VERIFY-}"
   exit 0
 fi
 
-OS_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${OS_ROOT}/hack/lib/init.sh"
-
-cd "${OS_ROOT}"
```

Comment:
Another place where you just broke running `../hack/verify-gofmt.sh`. Or maybe you meant for it to behave this new way...